app/services/encryptor.rb

```diff
@@ -0,0 +1,15 @@
+class Encryptor
+  ENCRYPTOR = ActiveSupport::MessageEncryptor.new(Rails.application.secrets.secret_key_base[0..31])
+
+  def self.encrypt(text)
+    encrypted_data = ENCRYPTOR.encrypt_and_sign(text)
+
+    CGI.escape(encrypted_data)
+  end
+
+  def self.decrypt(text)
+    encrypted_data = CGI.unescape(text)
+
+    ENCRYPTOR.decrypt_and_verify(encrypted_data)
+  end
```

Comment:
From IRL review: add test to see if `encrypt(x) == decrypt(encrypt(x))`

Reply:
Done!